Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM centos:centos7
+
+EXPOSE 8080
+
+ADD sendEmail-epel-7.repo /etc/yum.repos.d/
+
+RUN yum install -y epel-release && \
+    INSTALL_PKGS="httpd nagios telnet supervisor python-jinja2 nagios-plugins-all sendEmail perl-Net-SSLeay perl-IO-Socket-SSL" && \
+    yum -y --setopt=tsflags=nodocs install $INSTALL_PKGS && \
+    rpm -V $INSTALL_PKGS && \
```

Comment:
@rhcarvalho I see your point about the fatter layer because of the update. I have changed this now to be more like https://github.com/sclorg/postgresql-container/blob/master/9.5/Dockerfile, including the rpm -V to check the packages are installed.